src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -260,36 +255,37 @@ def __init__(self, schema_str, from_dict=None):
 
     def __call__(self, value, ctx):
         """
-        Deserializes Schema Registry formatted JSON to JSON object literal(dict).
+        Deserialize a JSON formatted record with Confluent Schema Registry framing to
+        a dict, or object instance according to from_dict, if from_dict is specified.
 
         Args:
-            value (bytes): Confluent Schema Registry formatted JSON bytes
+            value (bytes): A JSON serialized record with Confluent Schema Regsitry framing.
 
-            ctx (SerializationContext): Metadata pertaining to the serialization
-                operation.
+            ctx (SerializationContext): Metadata pertaining to the serialization operation.
```

Comment:
old nit: "pertaining".. as a non-native english speaker this is a word I never see or use.